CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+### v0.50.0
+###### Features
+- Update rule [E2507](https://github.com/aws-cloudformation/cfn-python-lint/blob/main/docs/rules.md#E2507) to validate resource configuration in an IAM policy (pull #[2023](https://github.com/aws-cloudformation/cfn-lint/pull/2023))
+- Update `aws-sam-translator` to `1.36.0` (pull #[2027](https://github.com/aws-cloudformation/cfn-lint/pull/2027))
+###### CloudFormation Specifications
+- Update resource specs to `37.1.0` (pull #[2012](https://github.com/aws-cloudformation/cfn-python-lint/pull/2012))
+- Update Lambda EOL for `dotnetcore2.1` (pull #[2015](https://github.com/aws-cloudformation/cfn-lint/pull/2015))
```

Comment:
trying to sneak in a few more of these Lambda EOL updates as well:
https://github.com/aws-cloudformation/cfn-lint/pull/2033